types/algoliasearch-helper/index.d.ts

```diff
@@ -6,6 +6,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
+/// <reference types="node" />
```

Comment:
it's actually the "events" package on npm, not the node builtin here, but it reminds me to remove it from DT, since it's not vendored in our own repository

Reply:
Ah. Whoops! Revised. Thank you!